packages/react/src/index.ts

```diff
@@ -29,16 +29,30 @@ export type {
 
 export type { SocketConfig } from './lib/useVoiceClient';
 
-export type {
-  AssistantEnd,
-  AssistantMessage,
-  AudioInput,
-  AudioOutput,
-  ChatMetadata,
-  SubscribeEvent,
-  ToolCallMessage,
-  ToolErrorMessage,
-  ToolResponseMessage,
-  UserInterruption,
-  UserMessage,
+import {
+  AssistantEnd as OriginalAssistantEnd,
+  AssistantMessage as OriginalAssistantMessage,
+  AudioInput as OriginalAudioInput,
+  AudioOutput as OriginalAudioOutput,
+  ChatMetadata as OriginalChatMetadata,
+  SubscribeEvent as OriginalSubscribeEvent,
+  ToolCallMessage as OriginalToolCallMessage,
+  ToolErrorMessage as OriginalToolErrorMessage,
+  ToolResponseMessage as OriginalToolResponseMessage,
+  UserInterruption as OriginalUserInterruption,
+  UserMessage as OriginalUserMessage,
 } from 'hume/api/resources/empathicVoice';
+
+type WithReceivedAt<T> = T & { receivedAt: Date };
```

Comment:
delete lines 15-28

rename types on 48-58 with the types that are exported in 15-28

Reply:
done